xml/common_intro_lifecycle_support_i.xml

```diff
@@ -124,7 +124,7 @@
   <itemizedlist>
    <listitem>
     <para>
-     Technology previews
+     Technology previews.
```

Comment:
According to the style guide, this should be 'technology previews' with no caps or period
https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-lists

Reply:
The problem with changing that is that this seems not to be widely observed -- so either this one list will be "correct" or I have the fairly epic job of changing all the bulleted lists in the manual. Or possibly all of the manuals. 

What do you reckon? Leave this instance but bear this in mind for the future, or start changing all of them? There's at least 1 other capitalized bullet-list in this sub-file alone.

Reply:
IMO stay focused on the task at hand and don't worry about global changes.

Reply:
Fixed.

Reply:
Thanks! For this new snippet in our prefaces, I copied an existing text from a bug IIRC and did not realize that the listitems did not match our style guide rules, my bad, sorry. I'll also fix this in the doc-kit repo which contains the boilerplate texts, otherwise the changes might be overwritten with an upddate from doc-kit in the future.